src/frontend/src/pages/WorkPackageForm/WorkPackageFormView.tsx

```diff
@@ -100,49 +113,23 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ wor
     remove: removeDeliverable
   } = useFieldArray({ control, name: 'deliverables' });
 
-  const {
-    data: project,
-    isLoading: projectIsLoading,
-    isError: projectIsError,
-    error: projectError
-  } = useSingleProject(validateWBS(projectWbsPipe(workPackage.wbsNum)));
-
-  const { mutateAsync } = useEditWorkPackage(workPackage.wbsNum);
-
-  if (allUsers.isLoading || !allUsers.data || projectIsLoading || !project) return <LoadingIndicator />;
-  if (allUsers.isError) {
-    return <ErrorPage message={allUsers.error?.message} />;
-  }
-  if (projectIsError) {
-    return <ErrorPage message={projectError?.message} />;
-  }
-
-  const blockedByToAutocompleteOption = (workPackage: WorkPackage) => {
-    return { id: wbsPipe(workPackage.wbsNum), label: `${wbsPipe(workPackage.wbsNum)} - ${workPackage.name}` };
-  };
-
-  const blockedByOptions =
-    project.workPackages.filter((wp) => wp.id !== workPackage.id).map(blockedByToAutocompleteOption) || [];
-
   const { userId } = user;
 
-  const users = allUsers.data.filter((u) => !isGuest(u.role));
-
   const transformDate = (date: Date) => {
     const month = date.getMonth() + 1 < 10 ? `0${date.getMonth() + 1}` : (date.getMonth() + 1).toString();
     const day = date.getDate() < 10 ? `0${date.getDate()}` : date.getDate().toString();
     return `${date.getFullYear().toString()}-${month}-${day}`;
   };
-  const onSubmit = async (data: WorkPackageEditFormPayload) => {
+  const onSubmit = async (data: WorkPackageFormViewPayload) => {
     const { name, startDate, duration, blockedBy, crId, stage } = data;
     const expectedActivities = mapBulletsToPayload(data.expectedActivities);
     const deliverables = mapBulletsToPayload(data.deliverables);
     const blockedByWbsNums = blockedBy.map((blocker) => validateWBS(blocker));
     try {
       const payload = {
-        projectLead: leadId ? parseInt(leadId) : undefined,
-        projectManager: managerId ? parseInt(managerId) : undefined,
-        workPackageId: workPackage.id,
+        projectLeadId: leadId ? parseInt(leadId) : undefined,
```

Comment:
hide the project manager and lead selects on create

Reply:
also I think it'd be good to display the wbsNumber in the crId field label, so the user knows what CR the wp they're creating is associated with:
<img width="348" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/4bafdeff-f03f-4da7-8796-def9e7105252">